### DISCUSSION TITLE
When release of BC-FJA 1.0.2.4 will be available in maven central

### DISCUSSION THREAD

**Discussion by nibexo** [bcgit/bc-java#1358] [Q&A] [answered]
Hi, when we can expect release of BC-FJA 1.0.2.4 to be available in maven? As far as I know this version is ready since november 2022 but is unavailable in maven central.

**Comment by dghgit** [comment]
It will be published to Maven central when it's certified. The current version is under test now (note this version is not quite the same as what is on the BC website as the noncert version).

**Reply by nibexo** [reply]
Thank you for your answer. However, when we can expect end these tests and public this new version?

**Comment by nibexo** [comment]
@dghgit  
Thank you for your answer. However, when we can expect end these tests and public this new version?

**Comment by dghgit** [comment]
When NIST are finished. At the moment it's taking from 6 to 15 months to do an update.

**Comment by dghgit** [comment, marked as answer]
1.0.2.4 is now available on Maven Central.
